BigBaseV2/src/memory/byte_patch.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+namespace memory
+{
+	class byte_patch
+	{
+	public:
+		template <typename TAddr>
+		byte_patch(TAddr address, std::remove_pointer_t<std::remove_reference_t<TAddr>> value)
```

Comment:
Ctor should prolly be private? Otherwise user may create an instance and it won't be added to the vector

Reply:
Yeah sounds good.

Reply:
Done ✔️